clients/native/examples/README.md

```diff
@@ -0,0 +1,48 @@
+# Nym Desktop Client Code Examples 
+
+This directory contains example code for javascript, python, and rust. Please note that **none of these examples are production-ready**, and are included really just to show how the client interacts with the Nym system. 
+
+> Make sure that you have an instance of the `nym-client` set up and running locally on port 1977 before trying to run any of these examples. 
```

Comment:
Could it be worth to mention that port `1977` is the default one, so that people wouldn't be confused trying to figure out how to make the client run on that particular port?